src/Repository/SkillRepository.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace AppBundle\Repository;
+
+use AppBundle\Entity\Adherent;
+use Doctrine\ORM\EntityRepository;
+
+class SkillRepository extends EntityRepository
+{
+    /**
+     * Finds all availebles skills for autocomplete that connected user has not yet chosen.
+     */
+    public function getSkillsForAutocomplete(string $term, Adherent $user)
+    {
+        $qbUserSkills = $this
+            ->createQueryBuilder('us')
+            ->select('us.name')
+            ->leftJoin('us.summary', 'cv')
+            ->andWhere('cv.member = :user')
```

Comment:
Done for `$query->getArrayResult()`.

We exclude all skills with the same name that user's skills to not appear on the list.